csrc/fusion_segmenter.h

```diff
@@ -414,6 +449,10 @@ class SegmentedFusion {
     SegmentedGroup* makeFusionInputGroup();
     SegmentedEdge* makeEdge(SegmentedGroup* from, SegmentedGroup* to, Val* val);
     void cleanUnused();
+    const std::unordered_map<SegmentedGroup*, int64_t> deterministic_groups_map()
```

Comment:
nitpick: I'm usually against returning const value...

Wondering if @wujingyue has some opinion here. :stuck_out_tongue_closed_eyes:

Reply:
I didn't put much thought into making the return value const.

I must have copied it from here.
https://github.com/NVIDIA/Fuser/blob/main/csrc/ir/container.h#L55-L64